ChangeLog.d/uniformize_bounds_checks.txt

```diff
@@ -0,0 +1,9 @@
+Bugfix
+   * Add additional bounds checks in ssl_write_client_hello() preventing
+     output buffer overflow if the configuration declared a buffer that was
+     too small.
```

Comment:
Isn't this redundant with the bugfix entry in `bad_user_configuration.txt`?

Reply:
Yes this is redundant with the bugfix entry in bad_user_configuration.txt and it belongs to this commit thus I've removed bad_user_configuration.txt.